jupyter-notebooks/analysis-ready-data/ard_1_intro_and_best_practices.ipynb

```diff
@@ -748,17 +934,43 @@
+      "text/plain": [
+       "<Figure size 1440x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
```

Comment:
These NDVI plots are a bit ugly, can we try and make them more visually appealing? I'd recommend:
1. Making them both the same size
2. Making the colourbar the same scale (i.e., making them both go from the same min and max, maybe each -1.0 to +1.0, to be consistent with how we define NDVI above?)
3. Setting NDVI as the colourbar title and having a more meaningful plot title. What are you showing? Yes, it's the NDVI, but of what?

Reply:
creating a new PR for this issue